Lesson30/Kadir/my-app/src/components/Exercise8List/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+
+export const Exercise8List = () => {
+
+    const sourceList = ["Kadir",
+    "Firat",
+    "Kubra",
+    "Ece",
+    "Mesut"
+    ];
+
+    const handleList = () => {
+        setItem(sourceList);  
+    };
+
+    const [item, setItem] = useState( sourceList );
```

Comment:
Since data is expected to be in a form of array, use plural form of the nouns used to name the state :) 
```suggestion
    const [items, setItems] = useState(sourceList);
```